#/():;#?!

```diff
@@ -0,0 +1 @@
+:H:e:l:l:o: :W:o:r:l:d:!
```

Comment:
Since each character to the right of a colon is printed, you just need to type `Hello World` and add a colon to the left of each character

```suggestion
:H:e:l:l:o: :W:o:r:l:d
```

Reply:
According to the wiki “:!” Ends the program.

Reply:
According to the wiki "!" ends the program. This means the colon to the left of it is not needed.

Reply:
Ohh